src/http/forward.h

```diff
@@ -20,6 +20,8 @@ typedef RefCount<Http::Message> MessagePointer;
 class Stream;
 typedef RefCount<Http::Stream> StreamPointer;
 
+class ContentLengthInterpreter;
```

Comment:
please move up, these should be alphabetical whenever possible to ease searching and patch merging when the header grows large.

Reply:
Done.